### PR TITLE
Add techdict namespace to arphilosophywiki T3780

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2836,6 +2836,10 @@
 			NS_FEATURED => 'Featured_Page',
 			NS_FEATURED_TALK => 'Featured_Page_talk',
 		),
+		'arphilosophywiki' => array(
```
Comment (paladox) [inline LocalSettings.php]: Need to add this to the define at the top too.

Comment (Amanda-Catherine) [reply LocalSettings.php]: @paladox ?

Comment (paladox) [reply LocalSettings.php]: For example "define( 'NS_TECH', 1600 );"

Comment (Amanda-Catherine) [reply LocalSettings.php]: Oh pfft I knew that... *slaps face with trout* (dealing with some personal issues right now so I’m not at my best)

**Pull request merged**